src/Evolution/Particles/MonteCarlo/Packet.hpp

```diff
@@ -32,6 +34,10 @@ struct Packet {
     momentum[2] = p_z_;
   }
 
+  // Species of neutrinos (in the code, just an index used to access the
+  // right interaction rates; typically 0=nu_e, 1=nu_a, 2=nu_x)
```

Comment:
Not a concrete suggestion, but we will probably want some easier way to index this. E.g. an `enum` or something with the tags order. Just something that we will want to think about in the future